src/Application/Application.ts

```diff
@@ -99,6 +99,25 @@ export class Application {
     if (args.verbose) {
       transports.console.level = 'debug';
     }
+
+    if (args.json) {
+      if (args._[0] == 'iq') {
+        this.logger.debug('Attempting to start application');
```

Comment:
I don't like how this is right now (lots of duplication), just gotta do some quick investigation how to discard all stdout output (until I want to output the json)